Formula/manticoresearch.rb

```diff
@@ -34,33 +50,23 @@ class Manticoresearch < Formula
   fails_with gcc: "5"
 
   def install
+    # ENV["DIAGNOSTIC"] = "1"
+    ENV["ICU_ROOT"] = Formula["icu4c"].opt_prefix.to_s
+    ENV["OPENSSL_ROOT_DIR"] = Formula["openssl"].opt_prefix.to_s
+    ENV["MYSQL_ROOT_DIR"] = Formula["mysql-client"].opt_prefix.to_s
+    ENV["PostgreSQL_ROOT"] = Formula["libpq"].opt_prefix.to_s
+
     args = %W[
-      -DCMAKE_INSTALL_LOCALSTATEDIR=#{var}
-      -DBoost_NO_BOOST_CMAKE=ON
-      -DWITH_ICU=OFF
-      -DWITH_ODBC=OFF
+      -DDISTR_BUILD=homebrew
```

Comment:
```suggestion
      -DDISTR_BUILD=#{tap.user}
```

Reply:
That is just a word 'homebrew', that is internal key inside manticore code. It was 'macosbrew', but I've changed it (both, here and in upstream), as it reflects that it is not linked to macos only.
So, it is *not* a variable, it should always be 'homebrew', otherwise build will be broken